config/gulp/sass.js

```diff
@@ -43,6 +43,15 @@ gulp.task('copy-vendor-sass', function () {
   return stream;
 });
 
+gulp.task('copy-dist-sass', function () {
```

Comment:
I wanted to have `copy-vendor-sass` be a dependency here, but gulp seems to *always* run task dependencies even if they've already been run during the gulp process' lifetime, so I left it out, because I didn't want to execute `copy-vendor-sass` multiple times during the doggone build process.

I guess another option might be to have `copy-dist-sass` mention `copy-vendor-sass` as a dependency, and then have `sass` depend of `copy-dist-sass` instead of `copy-vendor-sass`, but I don't like that either because copying the SASS to the dist directory is really orthogonal to compiling the SASS in general. Blerg.